hydra/clients.py

```diff
@@ -7,24 +7,31 @@
 class Client:
 
     def __init__(self, **kwargs):
-        self.id = kwargs.get('id')
```

Comment:
Don't change source lines ordering if it's not relevant to the code. It makes it difficult to review. Add new lines at the end of the constructor method. Also, to change the ordering do it in a separate commit.